src/components/HamburgerMenu/UseDimensions.js

```diff
@@ -0,0 +1,12 @@
+import { useEffect, useRef } from 'react';
+
+export default function UseDimensions(ref) {
```

Comment:
file name as well as hook name should start with lowercase, like this: useDimension